### ISSUE TITLE
In game

### ISSUE THREAD

**Issue by Dxnyy** [Krach64/lol-bot-leveling-free#7]
The bot doesnt upgrade I also put for upgrade a spell w,x,c,v but it doesnt work, and its normal the bot just suicide instead farm or somethjing else?

**Comment by Krach64** [comment]
I will check why ashe doesn't upgrade spell.
The bot is not enough intelligent to play vs human player.
He is coded only to leveling new account to lvl 30 in coop vs ia game in beginner mode. In this mode the nexus is always at the bottom left of the map.

**Comment by Dxnyy** [comment]
No, its not true. The bot ONLY start game in blind mote, not vs IA idk why
